### PR TITLE
fix: change url base domain

### LINKED ISSUE #204: Shared file URLs use default S3 hostname
## Description
The dataEntries data returned from the API uses default S3 hostname instead of xyz.terraso.org.

## Steps To Reproduce
1. Go to URL a group with files in it, i.e. https://app.staging.terraso.org/groups/academic-research-test
2. Click the download button beside a file

## Expected behavior
```
"dataEntries": {
    "edges": [
        {
            "node": {
                "id": "fc380b23-a22f-451a-8d6b-6134bad27806",
                "name": "name name here2",
                "description": "222222 blah blah Good file description",
                "resourceType": "csv",
                "url": "https://files.staging.terraso.org/f3952a22-9328-4f09-8980-ba70a0b58a87/terraso-a11y-test_1.csv?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=xxxxxxx%2Fus-east-2%2Fs3%2Faws4_request&X-Amz-Date=20221003T193358Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=xxxxxxx",
```

## Actual behavior
```
"dataEntries": {
    "edges": [
        {
            "node": {
                "id": "fc380b23-a22f-451a-8d6b-6134bad27806",
                "name": "name name here2",
                "description": "222222 blah blah Good file description",
                "resourceType": "csv",
                "url": "https://s3.us-east-2.amazonaws.com/files.staging.terraso.org/f3952a22-9328-4f09-8980-ba70a0b58a87/terraso-a11y-test_1.csv?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=xxxxxxx%2Fus-east-2%2Fs3%2Faws4_request&X-Amz-Date=20221003T193358Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=xxxxxxx",


```

### PR DESCRIPTION
**Pull request by JeancarloBarrios**
### Checklist
- [x] Corresponding issue has been opened

### Related Issues
Fixes #204

### CODE REVIEW

**Comment by JeancarloBarrios** [comment]
It requires a change on the environment variables `WEB_CLIENT_ENDPOINT=> WEB_CLIENT_DOMAIN` Where should I document this?

File: terraso_backend/config/settings.py [hunk 1]
```diff
@@ -211,7 +211,11 @@
     "TESTING_ENDPOINT": "/graphql/",
 }
 
-WEB_CLIENT_URL = config("WEB_CLIENT_ENDPOINT", default="")
+
+WEB_CLIENT_DOMAIN = config("WEB_CLIENT_DOMAIN", default="")
+WEB_CLIENT_URL = f"https://{WEB_CLIENT_DOMAIN}"
```
Comment (paulschreiber) [inline terraso_backend/config/settings.py:216]: Please support custom protocol and port (so local development works) per the example I sent you.

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: terraso_backend/config/settings.py (hunk 1 above)
Comment (JeancarloBarrios) [reply terraso_backend/config/settings.py]: done

**Review by paulschreiber — APPROVED** [review]
Looks good. Please deploy to staging to test.

**Comment by paulschreiber** [comment]
@JeancarloBarrios Please run `make format` and `make lint` to fix the formatting.

**Pull request closed without merging**